spec/index.adoc

```diff
@@ -2973,6 +3047,9 @@ then
     if res.new_certified_data ≠ NoCertifiedData:
       certified_data[M.receiver] = res.new_certified_data
 
+    if res.new_global_timer ≠ NoGlobalTimer:
```

Comment:
Should we always set it, without any conditions?

Reply:
While this does not matter now, this formulation is a bit more general as it would allow to distinguish whether a change was made or not. Changes to the certified variables are handled analogously.